src/utils/updateNotes.js

```diff
@@ -0,0 +1,14 @@
+import { db } from "../firebase";
+
+export const updateNotes = (notesData, uid) => {
+  try {
+    db.collection(`users/${uid}/notes`).doc(notesData.id).update({
+      title: notesData.title,
+      enteredNotes: notesData.enteredNotes,
+      isPinned: notesData.isPinned,
+      color: notesData.color,
+    });
+  } catch (e) {
```

Comment:
Suggestion: you can destructure notesData in place as well, so you dont have to repeat notesData

Reply:
Also add favicon

Reply:
will do that